app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -100,7 +99,7 @@
       app:layout_constraintEnd_toEndOf="parent">
 
       <TextView
-        style="@style/Subtitle1"
+        style="@style/Subtitle1ViewCenter"
         android:layout_marginBottom="24dp"
         android:gravity="center"
```

Comment:
Can this be removed per the latest style?

Reply:
Done.